Student/Student/Routes.swift

```diff
@@ -271,7 +271,14 @@ let routeMap: KeyValuePairs<String, RouteHandler.ViewFactory?> = [
     "/:context/:contextID/pages/new": nil,
     "/:context/:contextID/pages/:url": pageViewController,
     "/:context/:contextID/wiki/:url": pageViewController,
-    "/:context/:contextID/pages/:url/edit": nil,
+    "/:context/:contextID/pages/:url/edit": { url, params, userInfo in
```

Comment:
yeah, it does routing differently. In Teacher the RN routing stuff is primary & calls into these native factories, instead of in Student where the native routing just will return a helm (RN) view.